### ISSUE TITLE
[Bug] system animation settings are not respected

### ISSUE THREAD

**Issue by lvmm** [mozilla-mobile/fenix#13813]
## Steps to reproduce
prerequisites: system animations are off (system-developer options-xxxxx animation scale are set to 0)

switch between bookmarks and history view in a blank tab OR switch between tabs OR open a new tab

### Expected behavior
instantaneous action, no animations

### Actual behavior
transition is still animated

### Device information

* Android device: Caterpillar S61, android 9
* Fenix version: 68.11.0

**Comment by s-ankur** [comment]
Duplicate of #3597
